src/WebJobs.Script/Workers/Profiles/Conditions/HostPropertyCondition.cs

```diff
@@ -68,27 +67,27 @@ public bool Evaluate()
                 return false;
             }
 
-            _logger.LogDebug($"Evaluating HostPropertyCondition with name: {Name}, value: {value} and expression {Expression}");
+            _logger.LogDebug("Evaluating HostPropertyCondition with name: {name}, value: {value} and expression {expression}", Name, value, Expression);
```

Comment:
Should HostProperty enum be public ?

Reply:
I don't see why it is, I can make it private!